plugins/env/app/views/samson_env/_stage_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<fieldset>
+  <legend>
```

Comment:
Probably for another PR, but would be nice to have a collapsable read-only view of the globally-set and deploy group-set environment variables here too for visibility.

Reply:
make a BRE jira ;)

Reply:
😿